app/assets/javascripts/figgy_boot.es6

```diff
@@ -18,7 +18,6 @@ import BoundingBoxSelector from "bounding_box_selector"
 export default class Initializer {
   constructor() {
     this.server_uploader = new ServerUploader
-    this.collection_bag_uploader = new CollectionBagUploader
```

Comment:
This needs to be extracted and moved to a separate PR.

Reply:
These changes are now addressed with https://github.com/pulibrary/figgy/pull/3181